package.json

```diff
@@ -1,22 +1,36 @@
 {
   "name": "vex-js",
-  "version": "2.3.4",
-  "description": "Beautiful, functional, dialogs in Javascript",
-  "author": "Adam Schwartz <aschwartz@hubspot.com>",
+  "version": "3.0.0",
+  "description": "Beautiful, functional dialogs in vanilla JavaScript",
+  "contributors": [
+    "Adam Schwartz <aschwartz@hubspot.com>",
+    "Brendan Batliner <brendan@omnipointment.com>"
+  ],
   "license": "MIT",
-  "main": "js/vex.js",
+  "main": "dist/vex.js",
   "repository": {
     "type": "git",
     "url": "https://github.com/HubSpot/vex.git"
   },
+  "dependencies": {
```

Comment:
I agree. Did you mean the dependencies and the devDependencies, or just the dependencies?